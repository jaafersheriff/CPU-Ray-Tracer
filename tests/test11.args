firsthit simple.pov 640 480 320 300
