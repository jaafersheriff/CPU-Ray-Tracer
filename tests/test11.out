Pixel: [320, 300] Ray: {0 0 14} -> {0.001033 0.1251 -0.9921}
T = 12.92
Object Type: Sphere
Color: 1 0 1
